### PR TITLE
Fix bugs and clarify the 5-minute getting started guide

### PR DESCRIPTION
**Pull request by marcua**

### CODE REVIEW

File: simple_workflow/workflow.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
     "description": "Crawl a web page for an image and rate it.",
     "certifications": [],
     "sample_data_load_function": {
-        "path": "simple_workflow.load_sample_data.name"
+        "path": "simple_workflow.load_sample_data.load"
```
Comment (marcua) [inline simple_workflow/workflow.json:7]: wrong function name :)

**Review by thisisdhaas — COMMENTED** [review]
lgtm.

**Comment by HemilModi03** [comment]
@marcua I still get the same error, despite your solution,
https://github.com/b12io/orchestra/issues/438

**Comment by marcua** [comment]
@HemilModi03 Sorry about that! I'll bump the version of Orchestra once I fix the other issue you highlighted, and then both issues should be resolved!

**Pull request merged**